src/stores/game.js

```diff
@@ -208,7 +208,10 @@ export const useGameStore = defineStore('game', {
       this.lastEventPlayerChoosing = newGame.lastEvent?.pNum === this.myPNum ?? null;
       this.lastEventDiscardedCards = newGame.lastEvent?.discardedCards ?? null;
       this.waitingForOpponentToStalemate =
-        (newGame.lastEvent?.requestedByPNum === this.myPNum && !newGame.gameIsOver) ?? false;
```

Comment:
swapped the bespoke `requestedByPNum` with the `playedBy` value we use everywhere. Now need to check moveType